lowcar/devices/KoalaBear/KoalaBear.cpp

```diff
@@ -5,358 +5,346 @@
 #define CTRL_REG      0b000
 #define TORQUE_REG    0b001
 
-//Control Register options
-//Dead time
+// Control Register options
+// Dead time
 #define DTIME_410_ns  0b00
 #define DTIME_460_ns  0b01
 #define DTIME_670_ns  0b10
 #define DTIME_880_ns  0b11
 
-//Current sense gain
+// Current sense gain
 #define ISGAIN_5      0b00
 #define ISGAIN_10     0b01
 #define ISGAIN_20     0b10
 #define ISGAIN_40     0b11
 
-//Enable or disable motors
+// Enable or disable motors
 #define ENABLE        0b1
 #define DISABLE       0b0
 
-//Set VREF scaling out of 256
+// Set VREF scaling out of 256
 #define TORQUE        0x70
 
 uint16_t ctrl_read_data;
 uint16_t torque_read_data;
 
+volatile int32_t enc_a, enc_b;
+
 //********************************** labels for useful indices **********************************//
 typedef enum {
-	//params for the first motor
-	DUTY_CYCLE_A  = 0, //Student's desired speed between -1 and 1, inclusive
-	DEADBAND_A    = 1, //between 0 and 1, magnitude of duty cycle input under which the motor will not move
-	CURRENT_A     = 2,
-	PID_ENABLED_A = 3, //True if using PID control; False if using manual drive mode
-	PID_KP_A      = 4, //these three are the PID coefficients
-	PID_KI_A      = 5,
-	PID_KD_A      = 6,
-	ENC_A         = 7, //encoder position, in ticks
-	//same params for the second motor
-	DUTY_CYCLE_B  = 8,
-	DEADBAND_B    = 9,
-	CURRENT_B     = 10,
-	PID_ENABLED_B = 11,
-	PID_KP_B      = 12,
-	PID_KI_B      = 13,
-	PID_KD_B      = 14,
-	ENC_B         = 15
+    // params for the first motor
+    DUTY_CYCLE_A  = 0, // Student's desired speed between -1 and 1, inclusive
+    DEADBAND_A    = 1, // between 0 and 1, magnitude of duty cycle input under which the motor will not move
+    CURRENT_A     = 2,
+    PID_ENABLED_A = 3, // True if using PID control; False if using manual drive mode
+    PID_KP_A      = 4, // these three are the PID coefficients
+    PID_KI_A      = 5,
+    PID_KD_A      = 6,
+    ENC_A         = 7, // encoder position, in ticks
+    //same params for the second motor
+    DUTY_CYCLE_B  = 8,
+    DEADBAND_B    = 9,
+    CURRENT_B     = 10,
+    PID_ENABLED_B = 11,
+    PID_KP_B      = 12,
+    PID_KI_B      = 13,
+    PID_KD_B      = 14,
+    ENC_B         = 15
 } param;
 
 typedef enum {
-	MTRA = 1,
-	MTRB = 0,
+    MTRA = 1,
```

Comment:
Do you happen to know why this ordering is irregular?

Reply:
No, idk.... I'll reverse it